game/templates/game/main_page.html

```diff
@@ -0,0 +1,296 @@
+<!DOCTYPE html>
+{% load static %}
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Games</title>
+
+    <!-- Bootstrap Stylesheet -->
+    <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css"
+          integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous"/>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.0/dist/css/bootstrap.min.css"
+          integrity="sha384-B0vP5xmATw1+K9KRQjQERJvTumQW0nPEzvF6L/Z6nronJ3oUOFUFpCjEUQouq2+l" crossorigin="anonymous">
+	<link rel="stylesheet" href="http://code.ionicframework.com/ionicons/2.0.1/css/ionicons.min.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/bootstrap-slider/10.0.0/css/bootstrap-slider.min.css">
+
+    <!-- Static Stylesheet -->
+    <link rel="stylesheet" href="{% static 'game/css/main_detail_styles.css' %}">
+
+    <style>
+        @import url('https://fonts.googleapis.com/css2?family=Nunito:wght@800&family=Roboto:wght@500&display=swap');
+    </style>
+</head>
+<body>
+
+
+<!-- Navbar -->
+<nav class="navbar navbar-expand bg-danger">
+        <a class="navbar-brand text-white" href="#">
+            <img src="https://www.pngkey.com/png/full/272-2725717_computer-vision-computer-vision-icon-png-white.png"
```

Comment:
Could you please use template inheritance here?
https://docs.djangoproject.com/en/3.2/ref/templates/language/#template-inheritance

I see that you have similar pieces in both templates. After splitting, there should be 3 templates - base.html, main_page.html and game_detail_page.html

Reply:
Got it, I'll fix it